scss/bitstyles/components/stats.stories.mdx

```diff
@@ -0,0 +1,141 @@
+          <div class="o-card l-flex l-items-center">
+            <div class="l-flex--shrink-0 u-margin-s--right">
+              <div class="o-badge o-badge--danger u-padding-l">
+                <svg class="o-icon o-icon--l" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" aria-hidden="true" width="18" height="18">
+                  <path d="M2.78,32.83a6.07,6.07,0,0,1,8.58-8.59L45.71,58.59a6.07,6.07,0,0,0,8.58,0L88.64,24.24a6.07,6.07,0,1,1,8.58,8.59L54.29,75.76a6.07,6.07,0,0,1-8.58,0Z" fill-rule="evenodd"/>
+                </svg>
+                <span class="u-sr-only">Decreasing</span>
+              </div>
+            </div>
+            <div>
+              <dt class="u-font--medium u-h6 u-fg--gray-50">In storage</dt>
+              <dd class="u-h2 u-font--bold u-line-height--min">104</dd>
+            </div>
+          </div>
+          <div class="o-card l-flex l-items-center">
+            <div class="l-flex--shrink-0 u-margin-s--right">
+              <div class="o-badge o-badge--gray u-padding-l">
+                <svg class="o-icon o-icon--l" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" aria-hidden="true" width="18" height="18">
+                  <path d="M32.83,97.22a6.07,6.07,0,1,1-8.59-8.58L58.59,54.29a6.07,6.07,0,0,0,0-8.58L24.24,11.36a6.07,6.07,0,1,1,8.59-8.58L75.76,45.71a6.07,6.07,0,0,1,0,8.58Z" fill-rule="evenodd"/>
+                </svg>
+                <span class="u-sr-only">Unmoving</span>
+              </div>
+            </div>
+            <div>
+              <dt class="u-font--medium u-h6 u-fg--gray-50">In transit</dt>
+              <dd class="u-h2 u-font--bold u-line-height--min">32</dd>
+            </div>
+          </div>
+          <div class="o-card l-flex l-items-center">
+            <div class="l-flex--shrink-0 u-margin-s--right">
+              <div class="o-badge o-badge--positive u-padding-l">
+                <svg class="o-icon o-icon--l" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" aria-hidden="true" width="18" height="18">
+                  <path d="M97.22,67.17a6.07,6.07,0,1,1-8.58,8.59L54.29,41.41a6.07,6.07,0,0,0-8.58,0L11.36,75.76a6.07,6.07,0,0,1-8.58-8.59L45.71,24.24a6.07,6.07,0,0,1,8.58,0Z" fill-rule="evenodd"/>
+                </svg>
+                <span class="u-sr-only">Increasing</span>
+              </div>
+            </div>
+            <div>
+              <dt class="u-font--medium u-h6 u-fg--gray-50">Submitted</dt>
+              <dd class="u-h2 u-font--bold u-line-height--min">379</dd>
+            </div>
+          </div>
+        </dl>
+      </div>
+    `}
+  </Story>
+</Canvas>
+
+As in this next example, the change indicators could be replaced by static iconography that improve the recognition of what’s being measured, instead of adding context. icon colors here should either all be the same, or if the statistics or categories have already assigned recognisable colors, aid recognition by using those here too.
```

Comment:
```suggestion
As in this next example, the change indicators could be replaced by static iconography that improve the recognition of what’s being measured, instead of adding context. Icon colors here should either all be the same, or if the statistics or categories have already assigned recognisable colors, aid recognition by using those here too.
```